src/QandA/components/QuestionItem.jsx

```diff
@@ -1,19 +1,23 @@
 import React from 'react';
+import AnswerList from './AnswerList.jsx';
 
-const QuestionItem = (props) => {
+const QuestionItem = ({question}) => {
   return (
-    <div className="question-item">
-      <div className="question-body">
-        <span className="question-icon">Q: </span>{props.question.question_body}
-      </div>
-      <div className="question-interaction interaction">
-        <div className="helpful-btn btn">
-          Helpful btn
+    <div className="question-and-answer-container">
+      <div className="question-item">
+        <div className="question-body">
+          <span className="question-icon">Q: </span>{question.  question_body}
```

Comment:
I'm a little unclear here if the space after the . is a mistake or some other unexpected feature I'm not seeing? any clarity on this?